models/order/src/order-from-quote-draft/generator.ts

```diff
@@ -0,0 +1,26 @@
+import { Reference } from '@commercetools-test-data/commons';
+import {
+  fake,
+  Generator,
+  oneOf,
+  sequence,
+} from '@commercetools-test-data/core';
+import { orderState, paymentState, shipmentState } from '../constants';
+import { TOrderFromQuoteDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/orders#orderfromquotedraft
+
+const generator = Generator<TOrderFromQuoteDraft>({
+  fields: {
+    version: sequence(),
+    quote: fake(() => Reference.random().typeId('quote')),
+    quoteStateToAccepted: fake(() => false),
```

Comment:
Why not a fake boolean that could be true?

Reply:
you can specify it as true when you call the function, it's probably not a good idea to default all drafts to generate accepted quotes?

Reply:
Docs say the default is `false` so I rolled with it, but I did think about using the fake boolean, pinky swear.

Reply:
Fixed in [a663e4d](https://github.com/commercetools/test-data/pull/476/commits/a663e4d6e812397deb5768129d16ba746b65fc6c)